.yarnrc.yml

```diff
@@ -0,0 +1 @@
+yarnPath: '.yarn/releases/yarn-1.23.0-20210726.1745.cjs'
```

Comment:
can you remove this and the `.yarn/releases` file please?

Reply:
sorry, `commit -am` got me.  you should consider checking in `.yarn/releases` and an rc file though, it makes development very difficult for those of us who have upgraded yarn to 2+.